applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -382,16 +415,35 @@ def _CreateLevelSetConvectionProcess(self):
                 raise Exception("The BFECC level set convection requires the Kratos ConvectionDiffusionApplication compilation.")
         else:
             linear_solver = self._GetLinearSolver()
+            if self._eulerian_error_compensation: # this is used to perform one step BFECC for the eulerian LS convection
+                bfecc_order = 1
+            else:
+                bfecc_order = 0
```

Comment:
I'd use a `bool` for this.

Reply:
I was first reserved it for future improvements of BFECC, now, I think it can be only true/false. So, I will change it.